gslib/tests/test_cp.py

```diff
@@ -2344,7 +2345,7 @@ def test_cp_minus_e(self):
         expected_status=1)
     self.assertIn('Copying file', stderr)
     self.assertIn('Skipping symbolic link', stderr)
-    self.assertIn('CommandException: No URLs matched: %s' % fpath2, stderr)
+    self.assertIn(NO_URLS_MATCHED_TARGET % fpath2, stderr)
```

Comment:
Nit: IMO it's better to have literal strings in tests. It improves readability and makes the tests more robust. In case the value of constant changes accidentally, this test will let you know that something has changed.

Reply:
I'm curious about the best choice, so I made a YAQs post:
http://yaqs/4676980226785280

I'll change it for now, though.

Reply:
Thanks, I would be curious to know what others think as well. YAQs post is a good idea. You might want to add a little more context there that the constant is coming from the actual code. 
I feel it is okay to use a constant that is separate for the test code, so that you don't have to repeat the literal everywhere in the test code. 
Another alternate idea would be to have a unittest to test the literal as well, then we can be sure that the literal is what we are expecting.